packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -151,7 +151,7 @@ class VmServiceWrapper implements VmService {
   void dispose() => _vmService.dispose();
 
   @override
-  Future evaluate(
+  Future<dynamic> evaluate(
```

Comment:
Prefer Future<Object> if the type system allows it.

Reply:
done